testing/MLDB-1253_concat_test.py

```diff
@@ -0,0 +1,141 @@
+#
+# MLDB-1253_concat_test.py
+# Mich, 2016-01-13
+# Copyright (c) 2016 Datacratic Inc. All rights reserved.
+#
+
+import json
+import traceback
+
+if False:
+    mldb = None
+
+_test_cases = []
+
+
+def log(thing):
+    mldb.log(str(thing))
+
+
+def perform(*args, **kwargs):
+    res = mldb.perform(*args, **kwargs)
+    assert res['statusCode'] in [200, 201], str(res)
+    return res
+
+
+def query(query):
+    res = perform('GET', '/v1/query', [['q', query], ['format', 'table']])
+    return json.loads(res['response'])
+
+
+def record_row(ds_url, row_name, cols):
```

Comment:
Did you see that we have a dataset object wrapped with methods to addRows and commit? example: https://github.com/mldbai/mldb/blob/master/testing/MLDB-878_experiment_proc.py#L8